normandy/recipes/api/views.py

```diff
@@ -6,14 +6,15 @@
 from rest_framework.exceptions import NotFound
 from rest_framework.response import Response
 
+from normandy.base.api.mixins import RevisionMixin
 from normandy.base.api.permissions import AdminEnabled
 from normandy.base.api.renderers import JavaScriptRenderer
 from normandy.recipes.models import Action
 from normandy.recipes.api.permissions import NotInUse
 from normandy.recipes.api.serializers import ActionSerializer
 
 
-class ActionViewSet(viewsets.ModelViewSet):
+class ActionViewSet(RevisionMixin, viewsets.ModelViewSet):
```

Comment:
This mixin shouldn't be able to affect the update method of this class, since that class never calls `super()` (praise be to Python 3). That being said, I don't know how the tests are passing.